scripts/semconv/changelog-gen.js

```diff
@@ -324,9 +324,7 @@ function semconvChangelogGen(aVer=undefined, bVer=undefined) {
   }
   const changelogEntry = [`
 * feat: update semantic conventions to ${bSemconvVer} [#NNNN]
-  * Semantic Conventions ${bSemconvVer}:
-    [changelog](https://github.com/open-telemetry/semantic-conventions/blob/main/CHANGELOG.md#${slugify(bSemconvVer)}) |
-    [latest docs](https://opentelemetry.io/docs/specs/semconv/)
+  * Semantic Conventions ${bSemconvVer}: [changelog](https://github.com/open-telemetry/semantic-conventions/blob/main/CHANGELOG.md#${slugify(bSemconvVer)}) | [latest docs](https://opentelemetry.io/docs/specs/semconv/)
```

Comment:
I changed this to put these all on one line, because GitHub Markdown renders the newlines, which looks unpleasing when rendered. For example, here is a rendering when broken onto multiple lines:

![Screenshot 2025-02-06 at 1 51 55 PM](https://github.com/user-attachments/assets/dfb5b146-3c59-4485-8274-3f38acce0927)